Libraries/LibWeb/FileAPI/FileReader.h

```diff
@@ -101,6 +102,12 @@ class FileReader : public DOM::EventTarget {
 
     static WebIDL::ExceptionOr<Result> blob_package_data(JS::Realm& realm, ByteBuffer, FileReader::Type type, Optional<String> const&, Optional<String> const& encoding_name);
 
+    void queue_a_task(GC::Ref<GC::Function<void()>>);
```

Comment:
with this name, it seems like it's unlikely future editors of this class will notice that this is not calling HTML::queue_a_task unless they really dig in. Fetch has "queue_fetch_task" (which is specced, iirc). Is there a way we can make this harder to misuse or misunderstand?

Seems suitable for a follow-up though, the code as is works fine :)

Reply:
Also feels like this should be some feature of "task queues" or task sources or ... parallel queues or w/e when they are called out as "such and such object has a queue".